crates/common/src/phase0/constants.rs

```diff
@@ -0,0 +1,21 @@
+use alloy_primitives::{aliases::B32,FixedBytes};
```

Comment:
```suggestion
use alloy_primitives::{B32,FixedBytes};
```

you should just be able to import B32 directly, let me know if that works or not